sw/airborne/firmwares/rotorcraft/main.c

```diff
@@ -218,6 +218,9 @@ STATIC_INLINE void main_init(void)
   // send body_to_imu from here for now
   AbiSendMsgBODY_TO_IMU_QUAT(1, orientationGetQuat_f(&imu.body_to_imu));
 #endif
+
+  // Do a failsafe check first
+  failsafe_check();
```

Comment:
This change is not related to the actuator driver, but is a more generic problem that in paparazzi the initial signal sent to actuators is not the failsafe value but the driver-initial values.